pkg/konfluxgen/konfluxgen.go

```diff
@@ -448,6 +426,65 @@ func Generate(cfg Config) error {
 
 			}
 		}
+
+		buf := &bytes.Buffer{}
+
+		// add default integration test scenario with the stage policies
+		config := IntegrationTestConfig{
+			Name:            appKey,
+			ApplicationName: cfg.ApplicationName,
+			Contexts: []IntegrationTestContext{{
+				Name:        "application",
+				Description: "Application testing",
+			}},
+		}
+
+		if len(cfg.FBCImages) > 0 {
+			config.ECPolicyConfiguration = "rhtap-releng-tenant/fbc-stage"
+		} else {
+			config.ECPolicyConfiguration = "rhtap-releng-tenant/registry-standard-stage"
+		}
+
+		ecTestDir := filepath.Join(cfg.ResourcesOutputPath, ApplicationsDirectoryName, appKey, "tests")
+		if err := os.MkdirAll(ecTestDir, 0777); err != nil {
+			return fmt.Errorf("failed to create directory for %q: %w", appKey, err)
+		}
+
+		if err := enterpriseContractTestScenarioTemplate.Execute(buf, config); err != nil {
+			return fmt.Errorf("failed to execute template for EC test: %w", err)
+		}
+		if err := WriteFileReplacingNewerTaskImages(filepath.Join(ecTestDir, "ec-test.yaml"), buf.Bytes(), 0777); err != nil {
+			return fmt.Errorf("failed to write application file: %w", err)
+		}
+
+		buf.Reset()
+
+		// add integration test scenario for override snapshots with prod policies
+		config = IntegrationTestConfig{
+			Name:            fmt.Sprintf("%s-override-snapshot", appKey),
```

Comment:
kinda related to the above comment, I miss an EC reference in the name

Reply:
wenn we have the EC reference in the name, then the name will end up with something like `serverless-operator-135-override-snapshot-enterprise-contract` and will be hashed. That was also the reason, why I removed the suffix it above.

Reply:
what about short names? `ec` or `ovr` ?

Reply:
like `serverless-operator-135-override-snapshot-ec` or `serverless-operator-135-ec` ?